tutorials/using-cloud-vpn-with-cisco-asr/index.md

```diff
@@ -0,0 +1,744 @@
+
+This guide is not meant to be a comprehensive setup overview for the device referenced, but rather is only intended to assist in the creation of IPsec connectivity to Google Cloud Platform (GCP) VPC networks. The following is a high level overview of the configuration process which will be covered:
+
+*   Configure the base network configurations to establish L3 connectivity
+*   Setup the Base VPN configuration, including:
+    *   Configure IKEv2 Proposal and Policy
+    *   Configure IKEv2 Keyring
+    *   Configure IKEv2 profile
+    *   Configure IPsec Security Association (SA)
+    *   Configure IPsec transform set	
+    *   Configure IPsec profile
+    *   Configure IPsec Static Virtual Tunnel Interface (SVTI)
+    *   Configure Static or Dynamic Routing Protocol to route traffic into the IPsec tunnel
+*   Testing the IPsec connection
+*   Advanced VPN configurations
+
+### Getting started
+
+The first step in configuring your Cisco ASR 1000 for use with the Google Cloud VPN service is to ensure that the following prerequisite conditions have been met:
+
+The Cisco ASR 1000 Series Router IPsec application requires:
+
+1.  Advanced Enterprise Services(SLASR1-AES) or Advanced IP Services Technology Package License (SLASR1-AIS)
+1.  IPsec RTU license (FLASR1-IPsec-RTU)
+1.  Encryption HW module (ASR1002HX-IPsecHW(=) and ASR1001HX-IPsecW(=)) and Tiered Crypto throughput license which applies to ASR1002-HX and ASR1001-HX chassis only.
+
+For a detailed ASR 1000 Series Router license information, refer to the [ASR 1000 Routers Ordering Guide](http://www.cisco.com/c/en/us/products/collateral/routers/asr-1000-series-aggregation-services-routers/guide-c07-731639.html).
+
+### IPsec parameters
+
+For the Cisco ASR 1000 IPsec configuration, the following details will be used:
+
+|Parameter | Value|
+--------- |  -----
+|IPsec Mode | Tunnel mode |
+|Auth protocol | Pre-shared-key |
+|Key Exchange | IKEv2 |
+|Start | Auto |
+|Perfect Forward Secrecy (PFS) | Group 16 |
+|Dead Peer Detection (DPD) | 60 5 periodic |
+
+The IPsec configuration used in this guide is specified below:
+
+| Cipher Role | Cipher |
+| ------------| -------|
+| Encryption | esp-aes 256 esp-sha-hmac |
+| Integrity | sha256 |
+| Diffie-Hellman (DH) | group 16 |
+| Lifetime | 36,000 seconds (10 hours) |
+
+## Configuration – GCP
+
+### IPsec VPN using static routing
+
+This section provides the steps to create Cloud VPN on GCP. For a basic overview of what is Cloud VPN take a look at [this](https://cloud.google.com/compute/docs/vpn/overview) document. There are 2 ways to create VPN on GCP, using GCP console and the gcloud command-line tool. The upcoming section provide details to both in detail below:
+
+#### Using GCP console
+
+1.  [Go to the VPN page](https://console.cloud.google.com/networking/vpn/list?_ga=1.204702416.1585642068.1497031525) in the GCP Console. 
```

Comment:
We never call it the "GCP Console". Either "Google Cloud Platform Console" or "Cloud Platform Console".